### DISCUSSION TITLE
Elementary OS Screen becomes yellow

### DISCUSSION THREAD

**Discussion by Enri1403** [elementary/triage#411] [Q&A]
Hi, I usually use other distros but I saw Elementary OS and I decided it was worth giving a try! After the installation, I booted into Elementary OS and for a brief second, the screen was normally how it should be, but after that moment everything turned into a yellow tint.
![image](https://github.com/elementary/triage/assets/79421817/5b838d5f-50b0-4908-a49d-05f417c6333a)
That is how the screen looks like...
I think it's something about the color profile, I've already played with the color settings in the display category, but nothing was about color profiles and nothing worked.. I really like elementary OS and I wanna make it my daily Linux distro so please if anyone can help me it would be great

**Comment by jlnr** [comment]
I take it you are using an Nvidia GPU? Because this seems to be an issue with the free Nouveau graphics driver. When you install elementary OS, you have the option to install additional drivers (requires Wi-Fi/internet), that should automatically install the proprietary Nvidia driver.

I also have an Nvidia machine, and my next problem after the wrong colors the browser always showed blank pages. I had to install a browser from the terminal (e.g. `sudo apt install firefox`). Just bringing it up in case that you run into this issue next...
